spec/factories/profile_link_sites.rb

```diff
@@ -185,6 +185,55 @@
       validate_find '\A(https?://)?(www.)?(trakt.tv/)?(users/)?(?<username>[a-zA-z0-9_\-]+)\z'
       validate_replace 'https://trakt.tv/users/\k<username>'
     end
+    
+    # MyAnimeList
+    trait :myanimelist do
+      validate_find '\A(https?://)?(www.)?(myanimelist.net/)?(profile/)?(?<username>[a-zA-Z0-9_\-]+)\z'
+      validate_replace 'https://myanimelist.net/profile/\k<username>'
+    end
+    
+    # Annict
+    trait :annict do
+      validate_find '\A(https?://)?(www.)?(annict.com/)?(@)?(?<username>[a-zA-Z0-9]+)\z'
+      validate_replace 'https://annict.com/\k<username>'
+    end
+    
+    # MyFigureCollection
+    trait :myfigurecollection do
+      validate_find '\A(https?://)?(www.)?(myfigurecollection.net/)?(profile/)?(?<username>[a-zA-Z0-9]+)\z'
+      validate_replace 'https://myfigurecollection.net/profile/\k<username>'
+    end
+    
+    # Pixiv
+    trait :pixiv do
+      validate_find 'A(https?://)?(www.)?(pixiv.net/)?(member.php?id=)?(?<username>[0-9]+)\z'
+      validate_replace 'https://pixiv.net/member.php?id=\k<username>'
+    end
+    
+    # Booth.pm
+    trait :boothpm do
+      validate_find 'A(https://)?(?<username>[a-zA-z0-9]+)(.booth.pm)?\z'
```

Comment:
should be `\A` and also make sure it is `(https?://)?`, you forgot the `?` after the `s`. Check all the ones you did please, I have marked them as **here**.